components/bl-LightGallery/src/index.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+
+import Gallery from './Gallery';
+
+export default function MyCustomComponent({ component }) {
+  const [images, setImages] = useState([]);
+  const DEFAULT_ITEM_HEIGHT = 250;
+  const DEFAULT_SPEED = 400;
+
+  const options = {
+    itemHeight : +component.itemHeight || DEFAULT_ITEM_HEIGHT,
+    rotate : component.rotate === 'true',
+    thumbnail :component.thumbnail === 'true',
+    zoom : component.zoom === 'true',
+    autoplay : component.autoplay === 'true',
+    gap: +component.gap,
+    speed: +component.speed || DEFAULT_SPEED,
+    pager: component.pager === 'true',
+  };
+
+  useEffect(() => {
+    const newData = Object.assign({}, component.imagesData);
+    const imagesArray = [];
+
+    for(const key in newData) {
+      const point = Object.assign({}, newData[key]);
+      point.isDescription = point.hasOwnProperty('description');
```

Comment:
isDescription -> hasDescription

What value comes here if we don't set description in UI Builder?

I believe `hasDescription` is redundant. We could check for description where that needed just by looking into value of that property `(!!description)`